### DISCUSSION TITLE
Endpoint for GitHub API with Enterprise Repo

### DISCUSSION THREAD

**Discussion by ryanhebert1** [community/community#55955] [Enterprise]
### Select Topic Area

Question

### Body

Attempting to update the contents of a file via [documentation here](https://docs.github.com/en/rest/repos/contents?apiVersion=2022-11-28#create-or-update-file-contents) and unable to find a working Uri value. The endpoint requires the repo's created by user id, which this repo does not have as it is an enterprise account. Omitting this has not worked, nor has replacing it with the org name. Any suggestions? Is this possible?

**Comment by byrneh** [comment]
Hi I have used this API successfully using the Org Name and Repo name (it was under an enterprise account). 
If the file does exist it will create it.
Once created I needed to provide the additional SHA commít value to replace the existing FILE.
